test/test_minio_runner_binary_manager.rb

```diff
@@ -71,7 +71,7 @@ def test_that_it_redownloads_if_installed_and_version_cache_expired
     end
 
     # cache is determined by the mtime of the version file
-    `touch -d "2 days ago" #{TestBinary.version_file_path}`
+    `touch -d "#{(DateTime.now - 3).strftime("%Y-%m-%dT%H:%M:00")}" #{TestBinary.version_file_path}`
```

Comment:
The date format for `touch -d` is stricter on macOS so this test would fail here. We can just make it a format that works on both linux and macOS